src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,47 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String separator = System.lineSeparator();
+    private static final SalaryInfo salaryInfo = new SalaryInfo();
+    private static final int DATE_INDEX = 0;
+    private static final int HOURS_OF_WORK_INDEX = 2;
+    private static final int NAME_INDEX = 1;
+    private static final int SALARY_PER_HOUR_INDEX = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder stringBuilder = new StringBuilder("Report for period ");
+        stringBuilder.append(dateFrom)
+                .append(" - ")
+                .append(dateTo);
```

Comment:
let's think of purpose of this lines, as this piece of code has its responsibility we can create a separate method